assets/js/components/notifications/BannerNotifications.js

```diff
@@ -70,6 +72,7 @@ export default function BannerNotifications() {
 					{ ga4ActivationBannerEnabled && <ActivationBanner /> }
 				</Fragment>
 			) }
+			{ twgEnabled && <ThankWithGoogleSupporterWallNotification /> }
```

Comment:
Guess I wasn't too specific in the IB and may have missed this. But looking at this closely, this notification should also only be visible to admins who can modify TwG settings (not for view only shared dashboard users). So we should move this into the above `<Fragment>` after the `ga4ActivationBanner`.